moulinette/interfaces/cli.py

```diff
@@ -131,7 +127,7 @@ def pretty_print_dict(d, depth=0):
         - depth -- The recursive depth of the dictionary
 
     """
-    keys = d.keys()
+    keys = list(d.keys())
```

Comment:
To be confirmed (I don't know if "keys" is accessed using []) but I don't think it's needed.